libs/wingsdk/src/helpers.ts

```diff
@@ -1,14 +1,17 @@
 // Code in this file will be automatically included in all inflight code bundles,
 // so avoid importing anything heavy here.
-import { deepStrictEqual, notDeepStrictEqual } from "node:assert";
+import { notDeepStrictEqual } from "node:assert";
 import * as path from "node:path";
 import type { Construct } from "constructs";
+// since we moved from node:18 to node:20 the deepStrictEqual doesn't work as expected.
+// https://github.com/winglang/wing/issues/4444
+// therefore we're using a local version of the comparison from node 18: https://github.com/nodejs/node/blob/v18.x/lib/internal/util/comparisons.js
```

Comment:
This note should probably just be in `equality.ts` instead.

Reply:
changed it, the link appears now only in the `equality.ts` file